src/main/java/gregtech/api/capability/impl/HeatingCoilRecipeLogic.java

```diff
@@ -18,6 +22,36 @@ public class HeatingCoilRecipeLogic extends MultiblockRecipeLogic {
 
     public HeatingCoilRecipeLogic(RecipeMapMultiblockController metaTileEntity) {
         super(metaTileEntity);
+        if (!(metaTileEntity instanceof IHeatingCoil)) {
```

Comment:
Should this be before the super call?

Reply:
In java the super constructor must always be called first, so this has to go after it. It will cause a compilation error otherwise.